README.md

```diff
@@ -13,12 +13,17 @@
 
 `avail-light` is a data availability light client that can do the following:
 
-* Listen for newly produced blocks.
+* Listen for newly mined blocks
```

Comment:
I would prefer `forged` over `mined`. However, I think right now we listen to `finalized` blocks, right?